database_admin/migrations/042_delete_packages_old.up.sql

```diff
@@ -0,0 +1,36 @@
+DROP TABLE system_package;
```

Comment:
Ok, so you really joined phase 2 - renaming with 3 - deleting. But if you are ok with that....